src/s1reader/s1_burst_slc.py

```diff
@@ -622,3 +627,8 @@ def swath_name(self):
         '''Swath name in iw1, iw2, iw3.'''
         return self.burst_id.split('_')[1]
 
+    @property
+    def bbox(self):
+        '''Returns the bounding box of the burst.'''
+        # Use the provided shapely function to get the bounding box
```

Comment:
Could you write out in the comment the detailed format of the `border[0].bounds`, e.g. `list of 4 float, representing south, north, west, east in the unit of degrees`?

Reply:
added, although just confirming that the shapely `bounds` returns (west, south, east, north) rather than the isce2 box conventions

Reply:
Thanks.